### PR TITLE
Add mumax3c

### PR DESCRIPTION
**Pull request by lang-m**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by samjrholt** [comment]
I confirm I want to be a maintainer.

**Comment by fangohr** [comment]
I confirm I am happy to be a maintainer.

**Comment by swapneelap** [comment]
I confirm I want to be a maintainer.

**Comment by lang-m** [comment]
@conda-forge/help-python this recipe is ready for review!

File: recipes/mumax3c/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  requires:
+    - pip
+
+about:
+  home: http://ubermag.github.io/
+  summary: Python interface to mumax3 integrated into Jupyter notebook.
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  doc_url: https://ubermag.github.io/documentation/mumax3c.html
```
Comment (beenje) [inline recipes/mumax3c/meta.yaml:44]: This url returns 404. Could you please fix?

**Review by beenje — CHANGES_REQUESTED** [review]

File: recipes/mumax3c/meta.yaml (hunk 1 above)
Comment (lang-m) [reply recipes/mumax3c/meta.yaml:44]: The website is now online. (We initially intended to make it public once the package is available on conda-forge)

**Pull request merged**